lib/ex_doc/formatter/html/templates/index_template.eex

```diff
@@ -4,9 +4,18 @@
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
 	<title><%= config.project %> v<%= config.version %> Documentation</title>
+	<script type="text/javascript" charset="utf-8">
```

Comment:
This is awesome! Should we embed it in the page or can we add it to any of the javascript files?